include/supernode.h

```diff
@@ -71,8 +125,21 @@ class Supernode {
    * @param updates the batch of updates to apply.
    * @return
    */
-  static Supernode* delta_supernode(uint64_t n, long seed, const
+  static Supernode::SupernodeUniquePtr delta_supernode(uint64_t n, long seed, const
```

Comment:
This is (or at least should be) useless. Our graph workers have pre-allocated buffers for delta nodes. There should be no reason to need to allocate space on the fly like this.

Reply:
This was left in for testing purposes.

Reply:
Can you refactor this as a utility/helper function into our testing code then?

Reply:
I just removed it.